.rubocop.yml

```diff
@@ -210,3 +212,11 @@ Style/StringLiterals:
   EnforcedStyle: double_quotes
 Style/SwapValues:
   Enabled: true
+
+# TODO these were temporarily disabled during the Ruby 2.7 -> 3.1 upgrade
+# in order to keep the upgrade diff small, they will be enabled/fixed in
+# a follow-on PR.
+Style/MutableConstant:
+  Enabled: false
+Style/RedundantFreeze:
+  Enabled: false
```

Comment:
Opened https://github.com/dependabot/dependabot-core/pull/5468 to re-enable these checks and fix the existing violations.